pkg/cmd/run.go

```diff
@@ -685,6 +706,8 @@ func init() {
 	runCmd.Flags().String("mount-format", "json", fmt.Sprintf("file format to use. if not specified, will be auto-detected from mount name. one of %v", models.SecretsMountFormats))
 	runCmd.Flags().String("mount-template", "", "template file to use. secrets will be rendered into this template before mount. see 'doppler secrets substitute' for more info.")
 	runCmd.Flags().Int("mount-max-reads", 0, "maximum number of times the mounted secrets file can be read (0 for unlimited)")
+	runCmd.Flags().StringSliceVarP(&secretsToInclude, "only-secrets", "", []string{}, "only include the specified secrets")
```

Comment:
Since you're no longer using a short flag, you should use `StringSliceVar` instead